project/webapp/templates/static_templates/about.html

```diff
@@ -9,11 +9,11 @@
 </head>
 
 <script id="about-template" type="text/template">
-    <header>
+<header>
     <div class="content-header center">
         <div class="container section">
             <h2 class="title">About Us</h2>
-            <p class="description">The team that is bringing you CiviWiki</p>
+            <p class="description">The team that is bringing you civiwiki</p>
```

Comment:
I believe 'civiwiki' should be camel-case, i.e. 'CiviWiki'. At least it is camel-case in other places, such as the mission statement. In any case, we should be consistent.

Reply:
In effect, CiviWiki is the name of the organization and software, so should probably be camel-case.